example/pte_2mat.cpp

```diff
@@ -0,0 +1,188 @@
+//------------------------------------------------------------------------------
+// © 2025. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+
+// C headers
+#include <cmath>
+#include <cstdio>
+
+// C++ headers
+#include <iostream>
+#include <memory>
+#include <string>
+
+// This library contains portable utilities
+#include <ports-of-call/portability.hpp>
+
+// This contains logic for indexers
+#include <singularity-eos/base/indexable_types.hpp>
+// This contains useful tools for preventing things like divide by zero
+#include <singularity-eos/base/robust_utils.hpp>
+// The PTE closures
+#include <singularity-eos/closure/mixed_cell_models.hpp>
+// Needed to import the eos models
+#include <singularity-eos/eos/eos.hpp>
+
+// Number of equations of state to use. For this example, it's 2.
+constexpr std::size_t NEOS = 2;
+// Number of PTE solves to do
+constexpr std::size_t NTRIAL = 1;
+
+// Set the EOSs you want to use here.
+using EOS = singularity::Variant<singularity::SpinerEOSDependsRhoT>;
```

Comment:
Again, does it make sense to make a custom variant?

Reply:
What do you mean? This is a custom variant? Do you mean I should be just using the standard one? Making it custom is just for faster compile times.

Reply:
For some reason I thought it was the standard variant... my bad!